creatures.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=AnimeCreatures', 'root', 'password');
+
+function display($db)
+{
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $db->prepare('SELECT `name`, `movie`, `year`, `special_abilities`, `creepiness`, `img_dir` FROM `creatures`;');
```

Comment:
Consider putting your query into a variable and passing it into the function as a parameter. That way you can use the DB connection function for any kind of query!